### PR TITLE
Handling large documents guide

### LINKED ISSUE #310: Advise about big documents that should be split
It would be great if we can redirect users with big documents issues to a page that explain why and how to split big documents like books.

I [already talked to someone about this MeiliSearch limitation](https://github.com/meilisearch/MeiliSearch/issues/556#issuecomment-605333369) and why he should split its articles by paragraph in an issue.

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #310

### CODE REVIEW

**Review by Kerollmops — CHANGES_REQUESTED** [review]

File: learn/cookbooks/improving_relevancy_large_documents.mdx [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+title: Improving relevancy when working with large documents — Meilisearch documentation
+description: Use JavaScript with Node.js to split a single large and configure Meilisearch with a distinct attribute to prevent duplicated results.
```
[the remark this answers is not in the document]
Comment (guimachiavelli) [reply learn/cookbooks/improving_relevancy_large_documents.mdx:3]: oops, thanks for spotting that 🤦

**Comment by curquiza** [comment]
It's happening!! 🎉

**Comment by guimachiavelli** [comment]
Thanks for the feedback, @kero. I corrected the numbers for what Meilisearch considers a large document according to your recommendation (not really sure where I got the initial numbers, but I did think they were much larger than what I would have expected).

**Review by Kerollmops — DISMISSED** [review]
That's perfect! Thank you Gui 😃

**Review by Kerollmops — DISMISSED** [review]
LGTM 👌

**Pull request merged**